synapse_data_warehouse/synapse_event/tasks/V2.54.1__create_fileupload_event_task.sql

```diff
@@ -0,0 +1,17 @@
+use database {{database_name}}; --noqa: JJ01,PRS,TMP
+
+-- Step 1) Pause the upstream tasks
+alter task synapse_raw.refresh_synapse_warehouse_s3_stage_task suspend;
+alter task synapse_raw.fileupload_task suspend;
```

Comment:
Nit. I had the impression that we only need to suspend the root task. (Not something must to be changed)